docs/src/main/sphinx/connector/hive-azure.rst

```diff
@@ -2,14 +2,13 @@
 Hive connector with Azure Storage
 =================================
 
-The :doc:`hive` can be configured to query
-Azure Standard Blob Storage and Azure Data Lake Storage Gen2 (ABFS). Azure Blobs
-are accessed via the Windows Azure Storage Blob (WASB). This layer is built on
-top of the HDFS APIs and is what allows for the separation of storage from the
-cluster.
-
-Trino supports both ADLS Gen1 and Gen2. With ADLS Gen2 now generally available,
-we recommend using ADLS Gen2. Learn more from `the official documentation
+The :doc:`hive` can be configured to use Azure Data Lake Storage Gen2 to map big
+data analytics onto Azure Standard Blob Storage. Trino supports Azure Blob File
+System (ABFS) to access data in ADLS Gen2.
+
+While Trino supports ADLS Gen1 and WASB, we recommend migrating to ADLS Gen2, as
```

Comment:
Let's update these with links:

> Trino also supports [ADLS Gen1](https://learn.microsoft.com/en-us/azure/data-lake-store/data-lake-store-overview) and Windows Azure Storage Blob driver (WASB), but we recommend [migrating to ADLS Gen2](https://learn.microsoft.com/en-us/azure/storage/blobs/data-lake-storage-migrate-gen1-to-gen2-azure-portal), as ADLS Gen1 and WASB are legacy options that will be removed in the future.

Reply:
Thanks for the review. I have pushed changes to incorporate your feedback.